### PR TITLE
📊 Guinea worm certification and reported cases dataset

### PR DESCRIPTION
**Pull request by spoonerf**

### CODE REVIEW

File: etl/steps/data/garden/who/2023-06-29/guinea_worm.meta.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  - *source_certification
+  - *source_reported_cases
+tables:
+  guinea_worm:
+    variables:
+      year_certified:
+        title: Year country is certified free from guinea worm
+        unit: ''
+        description:
+        - *certification_description
```
Comment (Marigold) [inline etl/steps/data/garden/who/2023-06-29/guinea_worm.meta.yml:52]: This is not valid, a description must be a string, yet this is a list in YAML. (It's being used like lists in a couple of YAML files, but these are special cases and we'll soon be fixing them)

There are couple more issues in the file. Are you using VSCode? If yes, then you should try [this extension](https://owid.slack.com/archives/C0193RW5E2J/p1673860572514829) that will raise warnings if there's something wrong and help you fix it easily.

**Review by Marigold — APPROVED** [review]
Looks good, although variable descriptions are not working and must be fixed.

File: etl/steps/data/garden/who/2023-06-29/guinea_worm.meta.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+descriptions:
+  certification: &certification_description |
+    Certification...
```
[the remark this answers is not in the document]
Comment (spoonerf) [reply etl/steps/data/garden/who/2023-06-29/guinea_worm.meta.yml:3]: It is! I haven't used this fancy metadata method before so just experimenting a bit here. It's not showing up in the variable description in my local grapher admin, so not sure I am doing it right.

File: etl/steps/data/garden/who/2023-06-29/guinea_worm.meta.yml (hunk 1 above)
Comment (spoonerf) [reply etl/steps/data/garden/who/2023-06-29/guinea_worm.meta.yml:52]: I will try that!

Comment (Marigold) [reply etl/steps/data/garden/who/2023-06-29/guinea_worm.meta.yml:52]: (It should be written as `description: *certification_description`)

Comment (spoonerf) [reply etl/steps/data/garden/who/2023-06-29/guinea_worm.meta.yml:52]: A silly question but what is this type of text called? Is it specific to yml files?

**Pull request merged**